go.mod

```diff
@@ -14,3 +14,5 @@ require (
 	gopkg.in/macaroon.v2 v2.1.0
 	gopkg.in/yaml.v2 v2.2.2
 )
+
+go 1.13
```

Comment:
You probably want to drop this line, otherwise people might not be able to use the lib with a lower version of Go. It's annoying that Go always adds it even when it's not needed :/

Reply:
Good catch - fixed.